hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieLogFile.java

```diff
@@ -94,10 +94,16 @@ public HoodieLogFile rollOver(FileSystem fs) throws IOException {
         FSUtils.makeLogFileName(fileId, extension, baseCommitTime, newVersion)));
   }
 
-  public static Comparator<HoodieLogFile> getLogVersionComparator() {
+  public static Comparator<HoodieLogFile> getBaseInstantAndLogVersionComparator() {
     return (o1, o2) -> {
-      // reverse the order
-      return new Integer(o2.getLogVersion()).compareTo(o1.getLogVersion());
+      String baseInstant1 = o1.getBaseCommitTime();
```

Comment:
rename to getBaseInstantTime() (and also the backing variable, to be consistent?)